web/src/components/organisms/Project/Schema/hooks.ts

```diff
@@ -0,0 +1,172 @@
+import { useCallback, useMemo, useState } from "react";
+
+import { Model } from "@reearth-cms/components/molecules/Schema/types";
+import {
+  useGetModelsQuery,
+  useCreateModelMutation,
+  useCreateFieldMutation,
+  SchemaFiledType,
+  SchemaFieldTypePropertyInput,
+} from "@reearth-cms/gql/graphql-client-api";
+
+type Params = {
+  projectId?: string;
+  modelId?: string;
+};
+
+export default ({ projectId, modelId }: Params) => {
+  const [modelModalShown, setModelModalShown] = useState(false);
+  const [fieldModalShown, setFieldModalShown] = useState(false);
+
+  const { data, refetch } = useGetModelsQuery({
+    variables: { projectId: projectId ?? "", first: 100 },
+    skip: !projectId,
+  });
```

Comment:
Are we paginating? This looks like how reearth-web used to look when we didn't paginate and when pagination was a little broken.